web/src/components/system/HostnamePage.tsx

```diff
@@ -99,6 +99,7 @@ export default function HostnamePage() {
 
         <Form id="hostnameForm" onSubmit={submit}>
           {success && <Alert variant="success" isInline title={success} />}
+          {requestError && <Alert variant="warning" isInline title={requestError} />}
```

Comment:
Not using `error` here, which would be more precise, because it is not being used in other forms at the time of writing. As said in the PR description, we're aware that better form validation and error reporting is needed but they should be addresses all together once we find a way to do so.